community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchAcceptanceTest.scala

```diff
@@ -128,7 +128,7 @@ class MatchAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsTe
       """.stripMargin
 
     // WHEN
-    executeWithRulePlanner(query)
+    updateWithBothPlanners(query)
```

Comment:
We should move this test case to `Merge*AcceptanceTest` (node or relationship version).